Gems/AtomContent/Sponza/Tools/Project_Env.bat

```diff
@@ -74,23 +74,20 @@ echo     DCCSI_LOGLEVEL = %DCCSI_LOGLEVEL%
 IF "%DCCSI_MAYA_VERSION%"=="" (set DCCSI_MAYA_VERSION=2020)
 echo     DCCSI_MAYA_VERSION = %DCCSI_MAYA_VERSION%
 
-:: LY_PROJECT is ideally treated as a full path in the env launchers
+:: O3DE_PROJECT_PATH is ideally treated as a full path in the env launchers
 :: do to changes in o3de, external engine/project/gem folder structures, etc.
-IF "%LY_PROJECT%"=="" (
-    for %%i in ("%~dp0..") do set "LY_PROJECT=%%~fi"
+IF "%O3DE_PROJECT_PATH%"=="" (
+    for %%i in ("%~dp0..") do set "O3DE_PROJECT_PATH=%%~fi"
     )
-echo     LY_PROJECT = %LY_PROJECT%
-
-:: this is here for archaic reasons, WILL DEPRECATE
-IF "%LY_PROJECT_PATH%"=="" (set LY_PROJECT_PATH=%LY_PROJECT%)
-echo     LY_PROJECT_PATH = %LY_PROJECT_PATH%
+echo     O3DE_PROJECT_PATH = %O3DE_PROJECT_PATH%
 
 :: Change to root Lumberyard dev dir
-:: You must set this in a User_Env.bat to match youe engine repo location!
-IF "%LY_DEV%"=="" (set LY_DEV=C:\Depot\o3de-engine)
-echo     LY_DEV = %LY_DEV%
+IF "%O3DE_DEV%"=="" echo     ~ You must set O3DE_DEV in a User_Env.bat to match your local engine repo!
+IF "%O3DE_DEV%"=="" echo     ~ Using default O3DE_DEV=C:\Depot\o3de-engine
```

Comment:
I think you may ask system team about how to get the o3de engine path from a gem. I think there is a better way to do it instead of have your own batch file to define it.

Reply:
I already know how to do that actually and have been incrementally moving forward to a point we can fully remove the .bat files.  The real problem here, is that a asset gem that you want to use as a 'maya project' might be external, not nested under the engine, and you may want to specify which repo, branch and/or engine that you want to boostrap.

These .bat files launch maya and treat the gem as the rooted maya project, but the engine DCCsi also bootstraps access to various things.

In a future increment, we will have a launcher and we can just launch procedurally.

Reply:
Basically, we can provision these .bat files in other asset gems - these two nested in the engine are just examples but I want the same patterns in place in the core and external gems for familiarity.